bower.json

```diff
@@ -16,8 +16,8 @@
   ],
   "dependencies": {
     "bootstrap": "~3.0.0",
-    "angular": "",
-    "jquery": "",
+    "angular": "~1.5.5",
+    "jquery": "~2.2.3",
```

Comment:
@careyrp did we also need to update this version as well? 
@dmepham was this the exact angular version we needed? i know ryan found a ticket, it would be great to reference that here

Reply:
I just picked both of these versions because they were the versions installed in the AMI the last time this worked. let me know if there are better verisons to use and I can update it

Reply:
oh yeah @careyrp can you link us to that discussion you found online with the regression that caused us to roll back from the latest version?

Reply:
https://stackoverflow.com/questions/42895684/angularjs-version-1-6-change-from-success-method-to-then-method